[AzureMonitorDistro] update readme to describe how to configure/disable Azure Sdk logs (#43362)
Azure/azure-sdk-for-net: sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

Refer to [Drop an instrument](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/metrics/customizing-the-sdk#drop-an-instrument) for more examples.

#### Disable Azure SDK Logs

> By default, the Azure Monitor Distro collects [client library requests and responses to Azure services](https://learn.microsoft.com/dotnet/azure/sdk/logging#logging-using-azuremonitoropentelemetryaspnetcore).

Reviewer: ```suggestion
By default, the Azure Monitor Distro collects [logs from Azure SDKs](https://learn.microsoft.com/dotnet/azure/sdk/logging#logging-using-azuremonitoropentelemetryaspnetcore).
```

> These are visible in your application's console output as well as in the Application Insights Traces.

Reviewer: ```suggestion
These are exported to Application Insights TraceTelemetry table.
```

This can be configured or disabled in the appsettings.json.

- You can control the log level of individual sources
    ```json
    {
      "Logging": {
        "LogLevel": {

>           "Azure.Core": "Debug"

Reviewer 2: Just to be a safe side, in case someone copy-pastes without reading
```suggestion
          "Azure.Core": "Warning"
```

>           "Azure.Core": "Warning"

Reviewer: Is there more specific ones like Azure.EventHub etc?
